Documentation/Tutorials/send-overrides-sendevent.md

```diff
@@ -0,0 +1,159 @@
+# Sending Datastream overrides using sendEvent
+
+Datastream overrides let you define additional configuration settings for your datastreams, allowing you to trigger different datastream behaviors on a per event basis.
+
+## Prerequisites
+
+* [Getting Started](../getting-started.md)
+* [Send Event API Reference](../api-reference.md#sendevent)
+
+## Datastream ID override
+
+After configuring the datastream overrides, you can now send these overrides to the Edge Network through Mobile SDK. Sending the overrides using the `sendEvent` API is the final step of activating the datastream configuration overrides.
+
+The following examples demonstrate what a datastream ID override looks like:
+
+```brightscript
+data = {}
+data["xdm"] = {
+    "eventType": "page.view",
+    "sampleXDMKey": "value
+}
+
+data["data"] = {
+    "Customkey": "value"
+}
+
+' Adding datastreamIdOverride and datastreamConfigOverride to the event data
+data["config"] = {
+    "datastreamIdOverride": "<YOUR_DATASTREAM_ID>"
+}
+
+adbSendEventCallback = sub(context, result)
+' Handle callback with response from sendEvent
+end sub
+
+m.aepSdk.sendEvent(data, adbSendEventCallback, m)
+
+```
+
+## Datastream configuration override
+
+The following examples demonstrate what a datastream configuration override could look like:
+
+
+```brightscript
+data = {}
+data["xdm"] = {
+    "eventType": "page.view",
+    "sampleXDMKey": "value
+}
+
+data["data"] = {
+    "Customkey": "value"
+}
+
+' Adding datastreamConfigOverride to the event data
+data["config"] = {
+    "datastreamConfigOverride" : {
+    "com_adobe_experience_platform": {
+        "datasets": {
+        "event": {
+            "datasetId": "<YOUR_DATASET_ID>"
+        }
+        }
+    }
```

Comment:
Can we modify the indentation in order to improve the readability of the object?